libs/events/Makefile

```diff
@@ -1,9 +0,0 @@
-.PHONY: docs
-REPO:=github.com/tendermint/tendermint/libs/events
-
-docs:
-	@go get github.com/davecheney/godoc2md
-	godoc2md $(REPO) > README.md
-
-test:
-	go test -v ./...
```

Comment:
I'm guessing this file is just not used?

Reply:
Right. Specifically, we don't use the `docs` target at all, and our tests use `go test` directly. So this is basically pointless. Moreover, the output of `godoc2md` is strictly worse for the user than browsing to godoc.org. 

So this isn't _exactly_ a fix to a link, but it's obsoleted by one.